### ISSUE TITLE
DUA 52223 Goodney -Amend - Add 2017 data w/finder file - Add Collab (Northwestern) 

### ISSUE THREAD

**Issue by MonicaAdamsFoster** [MonicaAdamsFoster/Monica#26]
┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1114082886939012/1159068728087223) by [Unito](https://www.unito.io/learn-more)

**Comment by MonicaAdamsFoster** [comment]
Email in from Kayla with updated ES & Request letter. I made some minor changes and forwarded to Steph to review. She needs to read the collaboration information to see if it is ready to go to BWH to submit to ResDAC.

**Comment by MonicaAdamsFoster** [comment]
Also correspondence with Kayla and ResDAC reconfirming waiting to process request for updated data years on the amendment submitted previously so the collaboration can be added.

**Comment by MonicaAdamsFoster** [comment]
Email from Kayla, she did some changes & sent to Phil for final ok.

**Comment by MonicaAdamsFoster** [comment]
Waiting on Kayla for info. to amend to add collaboration.

**Comment by MonicaAdamsFoster** [comment]
Email from Kayla with docs sent to BWH to get to ResDAC.

**Comment by MonicaAdamsFoster** [comment]
Kate from BWH responded she will review and respond.

**Comment by MonicaAdamsFoster** [comment]
The letters need the letter head removed. Nothing should be signed. Do not need the "Collaboration" sig addenda need one for Mary individually. Need to add the Data Custodian doc from prior amendment into the DUA. Need to make sure there is a collab checklist for NW.

**Comment by MonicaAdamsFoster** [comment]
Email sent to all to hold any submission to ResDAC until I review and respond with details for finalizing documents

**Comment by MonicaAdamsFoster** [comment]
Updated all docs and discussed with Steph. Sent email to Steph & Kayla with Phil & Allison as Cc: asked them to review before I send to all the Cc: people from the other email.

**Comment by MonicaAdamsFoster** [comment]
Fwd the email with the ES attached as I had questions on if Phil & Barbara actually are on the DUA as Project Staff as not on the EPPE I have but I can't access the project on  EPPE. Also indicated that the Spec worksheet needs info added to support the finder file request.

**Comment by MonicaAdamsFoster** [comment]
Kayla responded with multiple questions in regards to the review and documents to be submitted. I responded to A. Hawke that what we sent is accurate and needs the finder file details and BWH should send to ResDAC & Cc: us.

**Comment by MonicaAdamsFoster** [comment]
Need final details for finder file from Phil - Steph emailed him. Then MAF will send final docs to BWH to submit to ResDAC. A. Hawke sent BWH an email letting them know I would be contacting them.

**Comment by MonicaAdamsFoster** [comment]
Email response from Phil. Steph needs to talk to Barbara.

**Comment by MonicaAdamsFoster** [comment]
Email to Steph to see if she connected with Barbara indicated I want to send in by end of the week to ResDAC. Kept Allison on as a Cc

**Comment by MonicaAdamsFoster** [comment]
Steph has reached out to Barbara Gladders & Nivi R. for input on this.

**Comment by MonicaAdamsFoster** [comment]
A.Hawke emailed for an update & Steph indicated she was busy & AH emailed Barbara who didn't realize we were waiting or what we wanted from her conversation with Steph. BG will get back to us asap.

**Comment by MonicaAdamsFoster** [comment]
Email with info in from Barbara - need to check in with Steph on what needs to be done to the docs to get this submitted to ResDAC.

**Comment by MonicaAdamsFoster** [comment]
Email from Steph for me to drop info in to worksheet. Responded I will get done first thing in am and get all docs sent to ResDAC.

**Comment by MonicaAdamsFoster** [comment]
I need to send to BWH who will send to ResDAC

**Comment by MonicaAdamsFoster** [comment]
Response from Kate w/question on letters & indicated she'll review and let us know

**Comment by MonicaAdamsFoster** [comment]
Email in from Kate @ BWH re: Marc or someone on the team now needs to submit to ResDAC as the documents have been approved by thier Institution for submission. She sent updated BWH request letter which I put in SP.

**Comment by MonicaAdamsFoster** [comment]
Sent email to PI's re: did the docs get sent to ResDAC by BWH? Asked for the confirmation email to be forwarded to me.

**Comment by MonicaAdamsFoster** [comment]
Sent follow up email to All re: I haven't seen a Cc for this going to ResDAC, asked for confirmation it was sent.

**Comment by MonicaAdamsFoster** [comment]
Response from Marc re: does he send. Call with Steph, I should submit since C. Reese & PI have given the ok. Emailed all to let them know and this way I can follow up the process for completion.

**Comment by MonicaAdamsFoster** [comment]
Emailed docs to ResDAC with request for Amendment. Included anyone who was on any of the documents. Wait for ResDAC auto response and tracking ID.

**Comment by MonicaAdamsFoster** [comment]
Email confirmation of receipt of request in from ResDAC

**Comment by MonicaAdamsFoster** [comment]
Mary M. sent docs to ResDAC and  created a new ticket!!!! Steph responded to Phil & Allison and asked them to contact her to clarify she shouldn't send anything and it is already in process.

**Comment by MonicaAdamsFoster** [comment]
Email in from ResDAC re: next steps they are taking on their end.

**Comment by MonicaAdamsFoster** [comment]
Email from ResDAC re: 2016 data requested in this amendment was already on the DUA. Had to update all the documents that go with the amendment. Also had to add Jon S. on the DUA and correct me on the letter and Steph updated/respond to comments on the ES

**Comment by MonicaAdamsFoster** [comment]
Email in from ResDAC:
1. Requested a copy of the NOGA from Dr. McDermott's AHA funded study (this possibly needs it's own DUA)
2. Provide additional information on the VQI derived data in detail (include patient & bene counts) 
2.a A finder file for SSNs will need to be submitted for generation of the crosswalk
3. Request letter & ES attached will comments and revisions for review and response

**Comment by MonicaAdamsFoster** [comment]
Phil & Steph had email correspondence. Phil added track changes to ES. Steph responded we need the NOA for Mary and Phil needs to justify the VQI linkage launguage.

**Comment by MonicaAdamsFoster** [comment]
Asked Steph if she has the ES that Phil mentioned

**Comment by MonicaAdamsFoster** [comment]
Steph sent me the ES and I saved to SP
